### ISSUE TITLE
&nbsp; not working

### ISSUE THREAD

**Issue by andzandz11** [wilkelab/ggtext#51]
&nbsp; not working

```r
df = mtcars

# Labels with 
labs = c("Within &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Budget")


p = ggplot(df, aes(x=hp, y=mpg, color = factor(gear))) +
  geom_point(stat="identity") +
  scale_color_manual(values =c("red","black","green"),
                     aesthetics = "color",
                     labels = labs) +
  theme(legend.position = "bottom",
        legend.text = element_markdown(size=8)
  )
p

```

**Comment by clauswilke** [comment]
Yes, I'm aware. If there was an easy way to fix this I'd have done it already. It's eventually going to be addressed by moving to the [sinab rendering engine,](https://clauswilke.com/sinab/) but realistically that's at least a year out.

**Comment by andzandz11** [comment]
I found a workaround: `labs = c("Within<span> </span><span> </span><span> </span><span> </span>Budget")`
Maybe you can include that for the meantime in the readme or somewhere else becuase this took me 2h to figure out. But maybe I am just slow and there is an easier way. On Google there was nothing though._

**Comment by clauswilke** [comment]
I'll leave this issue open, so people may find your workaround if they search issues.

**Comment by clauswilke** [comment]
Addressed in #53.

**Comment by kendavidn** [comment]
I now use this function thanks to @andzandz11 
 
``` r
nbsp_func <- function(n){
  paste0(rep("<span> </span>", times = n), collapse = "")
}

nbsp_func(5)
#> [1] "<span> </span><span> </span><span> </span><span> </span><span> </span>"
```
